src/Common/TimeZone.php

```diff
@@ -55,6 +55,6 @@ public static function tZIdentifiers() :array
      */
     public static function validate($tz) :string
     {
-        return (in_array($tz, static::tZIdentifiers()) === true) ? $tz : 'UTC';
+        return in_array($tz, static::tZIdentifiers()) ? $tz : 'UTC';
```

Comment:
The `in_array` will return bool value, and removing the `=== true` statement makes sense.